tests/Mock/FetchTransactionPayPalSuccess.xml

```diff
@@ -0,0 +1,116 @@
+      </return>
+      <transaction xmlns="" xsi:type="vin:Transaction">
+        <VID xmlns="" xsi:type="xsd:string">[TRANSACTION_REFERENCE]</VID>
+        <amount xmlns="" xsi:type="xsd:decimal">[AMOUNT]</amount>
+        <currency xmlns="" xsi:type="xsd:string">[CURRENCY]</currency>
+        <divisionNumber xmlns="" xsi:type="xsd:string">12345</divisionNumber>
+        <merchantTransactionId xmlns="" xsi:type="xsd:string">[TRANSACTION_ID]</merchantTransactionId>
+        <timestamp xmlns="" xsi:type="xsd:dateTime">2016-10-04T08:06:04-07:00</timestamp>
+        <account xmlns="" xsi:type="vin:Account">
+          <VID xmlns="" xsi:type="xsd:string">[CUSTOMER_REFERENCE]</VID>
+          <merchantAccountId xmlns="" xsi:type="xsd:string">[CUSTOMER_ID]</merchantAccountId>
+          <emailAddress xmlns="" xsi:type="xsd:string">test@example.com</emailAddress>
+          <emailTypePreference xmlns="" xsi:type="vin:EmailPreference">html</emailTypePreference>
+          <name xmlns="" xsi:type="xsd:string">Test Customer</name>
+        </account>
+        <sourcePaymentMethod xmlns="" xsi:type="vin:PaymentMethod">
+          <VID xmlns="" xsi:type="xsd:string">[PAYMENT_METHOD_REFERENCE]</VID>
+          <type xmlns="" xsi:type="vin:PaymentMethodType">PayPal</type>
+          <payPal xmlns="" xsi:type="vin:PayPal">
+            <returnUrl xmlns="" xsi:type="xsd:string">https://vimeo.dev/haggle/pro/vindiciapaypalomnipay/finalize</returnUrl>
+            <cancelUrl xmlns="" xsi:type="xsd:string">https://vimeo.dev/haggle/pro/vindiciapaypalomnipay/finalize</cancelUrl>
+            <requestReferenceId xmlns="" xsi:type="xsd:int">1</requestReferenceId>
+            <referenceId xmlns="" xsi:type="xsd:string">B-27J89747YL063522A</referenceId>
+          </payPal>
+          <billingAddress xmlns="" xsi:type="vin:Address">
+            <VID xmlns="" xsi:type="xsd:string">99988877766655544433322211100000</VID>
+            <postalCode xmlns="" xsi:type="xsd:string">[POSTCODE]</postalCode>
+            <country xmlns="" xsi:type="xsd:string">[COUNTRY]</country>
+          </billingAddress>
+          <merchantPaymentMethodId xmlns="" xsi:type="xsd:string">[PAYMENT_METHOD_ID]</merchantPaymentMethodId>
+          <sortOrder xmlns="" xsi:type="xsd:int">1</sortOrder>
+          <active xmlns="" xsi:type="xsd:boolean">1</active>
+        </sourcePaymentMethod>
+        <statusLog xmlns="" xsi:type="vin:TransactionStatus">
+          <status xmlns="" xsi:type="vin:TransactionStatusType">Authorized</status>
+          <timestamp xmlns="" xsi:type="xsd:dateTime">2018-06-15T08:06:05-07:00</timestamp>
+          <paymentMethodType xmlns="" xsi:type="vin:PaymentMethodType">PayPal</paymentMethodType>
+          <payPalStatus xmlns="" xsi:type="vin:TransactionStatusPayPal">
+            <token xmlns="" xsi:type="xsd:string">[TOKEN]</token>
+            <paypalEmail xmlns="" xsi:type="xsd:string">[PAYPAL_EMAIL]</paypalEmail>
```

Comment:
This is the spelling they actually return, right?

Reply:
yes